api/app/views/mno_enterprise/auth/unlocks/_form.html.haml

```diff
@@ -0,0 +1,13 @@
+= form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f|
+  = devise_error_messages!
+  %div
+    = f.email_field :email, :placeholder => "Email", :required => true, 'ng-model' => 'email', :class => 'form-control'
```

Comment:
Use Ruby 1.9 syntax for hash

https://github.com/bbatsov/ruby-style-guide#hash-literals
https://github.com/bbatsov/ruby-style-guide#no-mixed-hash-syntaces